### PR TITLE
import command

### PR DESCRIPTION
**Pull request by jburel**
fix import command noticed while reviewing import workflow before montpellier workshop

cc @pwalczysko

### CODE REVIEW

File: docs/import-cli.rst [hunk 1]
```diff
@@ -65,31 +65,38 @@ Note: When importing for another user using the CLI, the importer1 does not have
     
     ``$ ls /OMERO/in-place-import/FRAP``
 
-#.  Run ``$ cd /opt/omero/server/OMERO.server/bin``
+#.  Active the virtual environment where ``omero-py`` is installed or add it to ``PATH`` e.g.
```
Comment (pwalczysko) [inline docs/import-cli.rst]:
```suggestion
#.  Activate the virtual environment where ``omero-py`` is installed or add it to ``PATH`` e.g.
```

File: docs/import-cli.rst [hunk 2]
```diff
@@ -65,31 +65,38 @@ Note: When importing for another user using the CLI, the importer1 does not have
...
+
+    
+    ``$ export PATH=/opt/omero/server/venv3/bin:$PATH``
+
+#. Point ``OMERODIR`` to the location where the OMERO server is installed e.g.
```
Comment (pwalczysko) [inline docs/import-cli.rst]:
```suggestion
#.  Point ``OMERODIR`` to the location where the OMERO server is installed e.g.
```

this space fix is crucial for correct indentation of the line below

**Comment by pwalczysko** [comment]
Two typos, otherwise builds and reads fine

**Comment by jburel** [comment]
Merging. This will be needed for the workshop during community meeting

**Pull request merged**